<?xml version="1.0" encoding="UTF-8"?><record_update sys_domain="global" table="ecc_agent_script_include">
    <ecc_agent_script_include action="INSERT_OR_UPDATE">
        <active>true</active>
        <description/>
        <name>PuppetEnterpriseAPIException</name>
        <script><![CDATA[/** 

* A runtime custom exception that is thrown in the course of an Ansible Tower API invokation. 

*/ 

var PuppetEnterpriseAPIException = Class.create(); 

PuppetEnterpriseAPIException.prototype = Object.extend(new RESTAPIException(), { 

  type: 'PuppetEnterpriseAPIException' 

}); ]]></script>
        <sys_class_name>ecc_agent_script_include</sys_class_name>
        <sys_created_by>admin</sys_created_by>
        <sys_created_on>2018-03-25 16:13:02</sys_created_on>
        <sys_domain>global</sys_domain>
        <sys_domain_path>/</sys_domain_path>
        <sys_id>d1d38771db8193000b5936cb7c961973</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_name>PuppetEnterpriseAPIException</sys_name>
        <sys_overrides/>
        <sys_package display_value="CfgMgmt" source="x_snc_cfgmgmt">8aea7ef5db4193000b5936cb7c96197a</sys_package>
        <sys_policy/>
        <sys_scope display_value="CfgMgmt">8aea7ef5db4193000b5936cb7c96197a</sys_scope>
        <sys_update_name>ecc_agent_script_include_d1d38771db8193000b5936cb7c961973</sys_update_name>
        <sys_updated_by>admin</sys_updated_by>
        <sys_updated_on>2018-03-25 16:13:02</sys_updated_on>
    </ecc_agent_script_include>
</record_update>
